packages/prerelease-registry/functions/prs/[[path]].ts

```diff
@@ -0,0 +1,91 @@
+import { generateGitHubFetch } from "../gitHubFetch";
+import { getArtifactForWorkflowRun } from "../runs/[[path]]";
```

Comment:
It also seems like a strange architecture to have route files (i.e. `[[path]].ts` exporting helper functions...

Reply:
Yeah, same point as https://github.com/cloudflare/wrangler2/pull/504#discussion_r810690545.